sdk/core/src/main/java/com/klaviyo/coresdk/Klaviyo.kt

```diff
@@ -53,62 +58,110 @@ object Klaviyo {
      * (e.g. after a fresh login)
      *
      * @param phoneNumber Phone number for active user
+     * @return
```

Comment:
nit: I mean doesn't really matter but add a return description here since you have it on the other setters?

Reply:
You know I almost just took those out earlier, they were just auto added by android studio. The return type here is `Klaviyo` (i.e. `this`) to allow chaining methods. Do you think I should say that in the `@return` description of all the setXYZ methods?

Reply:
I don't really care outside of how it affects the auto-generated docs. If you generate with dokka and it looks ok, its fine IMO. Usually I tend to just go overly-verbose when I'm doing docstrings.

Reply:
I added @return comments, i agree, especially in this file which is like 99% of our API!